packages/drawer/src/BackdropBase.tsx

```diff
@@ -0,0 +1,17 @@
+import { surfaceOverlay60 } from '@littlespoon/theme/lib/colors/token'
+import styled from 'styled-components'
+
+export const BackdropBase = styled.div`
+  align-items: center;
+  background-color: ${surfaceOverlay60};
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+  /* TODO: Update z-index value? */
```

Comment:
What is the purpose of the todo comment?

Reply:
@remarkablemark  Do you have an standard for z-indexes? I set random value for this `z-index` and probably you want me to update it; that is the reason I put this `TODO` comment

Reply:
Gotcha. We can follow the material design pattern of using power of 2s. We could add an enum for z-index in `theme` package. @fjallkraken what do you think?

Reply:
We could use MUI's `z-index` values as a reference: https://mui.com/material-ui/customization/z-index

Reply:
That works as well, I'm open to that. @fjallkraken thoughts?

Reply:
Fixed in https://github.com/little-spoon-dev/design-system/pull/609/commits/31ace5627eade9ea720fd18257e09cdcdee39dc0